spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -284,6 +284,28 @@
       end
     end
 
+    context 'POST to auth correctly stores SP in session' do
+      before do
+        @user = create(:user, :signed_up)
+        @saml_request = saml_request(saml_settings)
+        @post_request = saml_post_auth(@saml_request)
+        @stored_request_url = @post_request.request.original_url +
+                              '?SAMLRequest=' +
+                              @saml_request # CGI.escape(@saml_request)
```

Comment:
Did you mean to leave this comment in here?

Reply:
nope! 🤦‍♂